ocfweb/docs/docs/staff/procedures/restarting-services.md

```diff
@@ -19,3 +21,27 @@ Planned restarts of hypervisors should also be announced on this blog, since
 restarting hypervisors can often take several minutes or more.
 
 [status]: http://status.ocf.berkeley.edu
+
+## Rebooting hypervisors
+
+Rebooting hypervisors is a slightly risky business. Hypervisors aren't
+guaranteed to always reboot without problems. Therefore, you shouldn't reboot
+them unless you can physically access the lab in case problems arise.
+Additionally, this risk is the reason (D)SM permission is normally required to
+reboot hypervisors.
+
+So you've gotten the necessary permission and made a post on the status blog
```

Comment:
Link to the status blog?

Reply:
See the text above?

Reply:
Sure, I do think making this a link wouldn't hurt though, you can just use the same reference as above, and it helps if someone doesn't read the rest of the sections. I do agree they should read those too, but in my opinion having an extra link is pretty harmless.